### PR TITLE
Removal of emphasis concept

### PR DESCRIPTION
**Pull request by libertymayc**
Design have decided to remove concept of 'emphasis' and merge into variants (medium - primary, high - secondary, low - tertiary) to simplify communication and improve consistency

Notable:  Brings in new 'emphasize' prop - specifically for components with status - which needs to be discussed with devs

### CODE REVIEW

**Comment by libertymayc** [comment]
Note to self: update emphasis mentions in docs

File: .changeset/calm-clocks-dress.md [hunk 1]
```diff
@@ -7,5 +7,5 @@ Tweak palette disabled token used in editable
 ```diff
 - --uitk-editable-background-disabled-low: var(--uitk-palette-interact-background-low)
 + --uitk-palette-interact-background-disabled-low
-+ --uitk-editable-background-disabled-low: var(--uitk-palette-interact-background-disabled-low)
++ --uitk-editable-tertiary-background-disabled: var(--uitk-palette-neutral-tertiary-background-disabled)
 ```
```
Comment (libertymayc) [inline .changeset/calm-clocks-dress.md:11]: @joshwooding what should we do for changesets like this?

**Review by sion-docs — APPROVED** [review]

File: .changeset/calm-clocks-dress.md (hunk 1 above)
Comment (joshwooding) [reply .changeset/calm-clocks-dress.md:11]: I believe what you've done here is fine. These changesets haven't been released so can be edited, and conflicting changesets in a release could cause confusion.

Comment (libertymayc) [reply .changeset/calm-clocks-dress.md:11]: okay cool, i need to go through the changesets and make sure all are aligned with this change :)

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 2]
```diff
@@ -57,98 +52,60 @@ For example, a Button is a component on which a user can perform an action, so y
...
-Some characteristic tokens have a concept of emphasis. Components can apply characteristics with emphasis to make them stand out on the page amongst the rest of the UI, or have less prominence so that other components appear exaggerated.
-
-An example of a characteristic token which has emphasis levels is `--uitk-editable-background-` which has the three levels defined, `low`, `medium`, and `high`. By default, the FormField component has been styled to use the `--uitk-editable-background-medium` token in a local variable:
-
-```css
-.uitkFormField {
-  --formField-background: var(--uitk-editable-background-medium);
-}
-```
+### Simple customisation using characteristics
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:55]:
```suggestion
### Simple customization using characteristics
```

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 3]
```diff
@@ -57,98 +52,60 @@ For example, a Button is a component on which a user can perform an action, so y
...
 
-Which, in the UITK theme, just makes the background white. Let's make it appear on a purple Panel for demo purposes:
+In the following example, we will show how to customise the Card component through characteristics. Currently, Card only has one variant, using tokens such as `---uitk-container-primary-background` all of it's styling purposes. Here's how it looks by default, shown on a purple Panel for demo purposes:
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:57]:
```suggestion
In the following example, we will show how to customize the Card component through characteristics. Currently, Card only has one variant, using tokens such as `---uitk-container-primary-background` all of its styling purposes. Here's how it looks by default, shown on a purple Panel for demo purposes:
```

File: packages/theme/stories/characteristics/characteristics.stories.mdx [hunk 4]
```diff
@@ -57,98 +52,60 @@ For example, a Button is a component on which a user can perform an action, so y
...
-  --uitkCard-borderWidth: 3px;
+.my-app .uitkCard {
+  --uitkCard-background: var(--uitk-accent-background);
+  --uitkCard-borderColor: var(--uitk-accent-borderColor);
+  --uitkCard-color: var(--uitk-accent-foreground);
 }
 ```
 
-Note the use of `-high` emphasis characteristic tokens provided by UITK that weren't used in Card by default, as well as raw values to further emphasize the component. Now, all we have to do to make Card have emphasis is pass in the `uitkEmphasisHigh` class name:
+Note how no classname is needed to be passed in, we have just redefined the Card's styling at a higher specificity, to produce the following styling:
```
Comment (joshwooding) [inline packages/theme/stories/characteristics/characteristics.stories.mdx:102]:
```suggestion
Note how no class name is needed to be passed in. We have just redefined the Card's styling at a higher specificity, to produce the following styling:
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — APPROVED** [review]

**Pull request merged**